docs/source/operations/howto/connect-organizations.rst

```diff
@@ -0,0 +1,98 @@
+*************************************
+How to link multiple Substra backends
+*************************************
+
+This guide shows you how to enable communications between two Substra backends to make them exchange assets.
+This can be achieved either at deployment time or when the bakcend is already deployed.
+
+Prerequisites
+=============
+
+You will need `Helm <https://helm.sh>`_.
+
+Updating your Helm values
+=========================
+
+For the purpose of this guide we will use a setup with three Substra backends as shown in :ref:`Figure 1 <figure-1>`.
+
+.. _figure-1:
+.. mermaid:: diagrams/howto-link-backends-channels.mmd
```

Comment:
Do you think a representation like this would be more explicit? 
<img width="503" alt="image" src="https://user-images.githubusercontent.com/90057687/193816573-4ad28c2e-bbec-46a7-a7bd-d8e564683593.png">

Reply:
I'm not sure if the orchestrator should be part of an organization or not

Reply:
I think adding an orchestrator here would be confusing since there is no Orchestrator in this how to and it's never involved in this kind of assets exchange.

Another issue I have with this diagram is that it could imply that we can have multiple backends per organization. While in theory this is possible it is completely untested and I am not sure it is possible.

Reply:
Yes that's true, maybe we could add a similar figure in a higher level page in the documentation, and indicate that if the user follows all the How-to guides, he will end up with this setup in place (but that's for later).